### PR TITLE
migration: fix role import

### PR DESCRIPTION
**Pull request by liam-fitzgerald**
Fix role import, cause of Galens mystery channels

### CODE REVIEW

File: desk/app/heap.hoon [hunk 1]
```diff
@@ -408,7 +408,7 @@
   |^
   =/  =perm:h
     :_  group.association
-    ?:(=(~ writers) ~ (silt (rap 3 %heap '-' (scot %p p.flag) '-' q.flag ~) ~))
+    ?:(=(~ writers) ~ (silt (rap 3 'imports/' (scot %p p.flag) '/' q.flag ~) ~))
```
Comment (arthyn) [inline desk/app/heap.hoon:411]: this one says imports the others say import

Comment (liam-fitzgerald) [reply desk/app/heap.hoon]: Fixed

File: desk/app/groups.hoon [hunk 2]
```diff
@@ -525,7 +526,7 @@
   ::
   ++  sect-for-flag
     |=  =flag:g
-    `sect:g`(rap 3 'imports-' (scot %p p.flag) '/' q.flag ~)
+    `sect:g`(rap 3 'imports/' (scot %p p.flag) '/' q.flag ~)
```
Comment (arthyn) [inline desk/app/groups.hoon:529]: is this the same?

**Review by arthyn — APPROVED** [review]
lgtm, one question

**Pull request merged**